packages/aws-cdk-lib/aws-codedeploy/lib/base-deployment-config.ts

```diff
@@ -54,6 +54,39 @@ export enum ComputePlatform {
   ECS = 'ECS',
 }
 
+/**
+ * Configuration for CodeDeploy to deploy your application to one Availability Zone at a time within an AWS Region.
+ */
+export interface ZonalConfig {
+  /**
+   * The period of time that CodeDeploy must wait after completing a deployment to an Availability Zone.
+   *
+   * Minimum value is 1 second or 0.
+   *
+   * @default - CodeDeploy starts deploying to the next Availability Zone immediately
+   */
+  readonly monitorDuration?: Duration;
+
+  /**
+   * The period of time that CodeDeploy must wait after completing a deployment to the first Availability Zone.
+   *
+   * Minimum value is 1 second or 0.
```

Comment:
Is this phrasing of `Minimum value is 1 second or 0` taken from our docs somewhere else? It sounds a little confusing and I had actually initially misinterpreted it. Could we update this to something like
```
* Accepted Values:
*  * 0
*  * Greater than or equal to 1
```